addons/hr/models/hr_work_location.py

```diff
@@ -12,5 +12,7 @@ class WorkLocation(models.Model):
     active = fields.Boolean(default=True)
     name = fields.Char(string="Work Location", required=True)
     company_id = fields.Many2one('res.company', required=True, default=lambda self: self.env.company)
-    address_id = fields.Many2one('res.partner', required=True, string="Work Address", check_company=True)
-    location_number = fields.Char()
+    location_type = fields.Selection([
+        ('home', 'Home'),
+        ('office', 'Office'),
+        ('other', 'Other')], string='Cover Image', default='office', required=True)
```

Comment:
"Cover Image"?

Reply:
From the frontend the user sees a choice of 3 icons, which represent the location type, while under the hood it's this. I don't really like this solution either but as it's a 1 to 1 mapping between type and icon we will leave it like that for now. In the future if we want more variety in icons we will add a field just for that